app/api/migrations/migrations/19-rename-uploads-to-files/specs/19-rename-uploads-to-files.spec.js

```diff
@@ -43,14 +43,19 @@ describe('migration rename-uploads-to-files', () => {
   });
 
   describe('when files already exists', () => {
+    beforeEach(async () => {
+      const collections = await testingDB.mongodb.listCollections().toArray();
+      if (!collections.find(c => c.name === 'files')) {
+        await testingDB.mongodb.createCollection('files');
+      }
+    });
+
```

Comment:
New mongo driver fails on `createCollection` of existing collections, so this prior check is now needed.